modules/virt-disable-bootsource-update.adoc

```diff
@@ -7,16 +7,18 @@
 [id="virt-disabling-bootsource-update_{context}"]
 = Disabling automatic boot source updates
 
-You can reduce the number of logs on disconnected environments or reduce resource usage by disabling the automatic imports and updates of pre-defined boot sources. Set the `spec.featureGates.enableCommonBootImageImport` field in the `HyperConverged` custom resource (CR) to `false`.
+Disabling automatic boot source imports and updates can be helpful to reduce the number of logs in disconnected environments or to reduce resource usage.
+
+To disable automatic boot source imports and updates, set the `spec.featureGates.enableCommonBootImageImport` field in the `HyperConverged` custom resource (CR) to `false` to disable automatic boot source updates.
 
 [NOTE]
 ====
-Custom boot sources are not affected by this setting.
+User-defined boot sources are not affected by this setting.
 ====
 
 .Procedure
 
-* Use the following command to disable automatic updates:
+* Use the following command to disable automatic boot source updates:
 +
 [source,terminal]
 ----
```

Comment:
Small suggestion: Maybe break this up with backslashes so the user does not have to scroll so far to see the whole command?

Reply:
Sounds good. Thanks Avital, I always forget about the force-carriage return` \`